packages/functional-tests/tests/oauth/signinTokenCode.spec.ts

```diff
@@ -0,0 +1,114 @@
+import { EmailHeader, EmailType } from '../../lib/email';
+import { test, expect } from '../../lib/fixtures/standard';
+
+test.describe('OAuth signin token code', () => {
+  function toQueryString(obj) {
+    return Object.entries(obj)
+      .map((x) => `${x[0]}=${x[1]}`)
+      .join('&');
+  }
+
+  /* Email for fake account */
+  let email = '';
+
+  /* Password for fake account */
+  const password = 'passwordzxcv';
+
+  /* eslint-disable camelcase */
+  const queryParameters = {
+    client_id: '7f368c6886429f19',
+    code_challenge: 'aSOwsmuRBE1ZIVtiW6bzKMaf47kCFl7duD6ZWAXdnJo',
+    code_challenge_method: 'S256',
+    forceUA:
+      'Mozilla/5.0 (Linux; Android 6.0; Nexus 5 Build/MRA58N) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/65.0.3325.181 Mobile Safari/537.36',
+
+    keys_jwk:
+      'eyJrdHkiOiJFQyIsImtpZCI6Im9DNGFudFBBSFZRX1pmQ09RRUYycTRaQlZYblVNZ2xISGpVRzdtSjZHOEEiLCJjcnYiOiJQLTI1NiIsIngiOiJDeUpUSjVwbUNZb2lQQnVWOTk1UjNvNTFLZVBMaEg1Y3JaQlkwbXNxTDk0IiwieSI6IkJCWDhfcFVZeHpTaldsdXU5MFdPTVZwamIzTlpVRDAyN0xwcC04RW9vckEifQ',
+    redirect_uri: 'https://mozilla.github.io/notes/fxa/android-redirect.html',
+    scope: 'profile https://identity.mozilla.com/apps/notes',
+  };
+  /* eslint-enable camelcase */
+
+  test.beforeEach(async ({ target }) => {
+    // The `sync` prefix is needed to force confirmation.
+    email = `sync${Math.random()}@restmail.net`;
+    await target.createAccount(email, password, {
+      // Important, must set this to be directed to /sign_in_token_code page
+      verificationMethod: 'email-otp',
+    });
+  });
+
+  test('verified - invalid token', async ({
+    page,
+    pages: { login, relier, signinTokenCode },
+  }) => {
+    // Go to 123 with specific query params
```

Comment:
The `tokenCode` experiment was removed, we should need the experiment params anymore.